src/components/BooksPage/OneBook.js

```diff
@@ -1,49 +1,62 @@
+  } = oneBook;
 
   return (
-      <div className ="oneBookContainer">
-            <div className="oneBookInformation">
-                <div className="oneBookInformationText">
-
-                <span className="bookActionSpan">{oneBook.action}</span>
-                <h2 className="bookTitleH2">{oneBook.title}</h2>
-                <span className="bookAuthorSpan">{oneBook.author}</span>
-
-                </div>
-
-                <div className="oneBookInformationOptions">
-                    <ul>
-                        <li>Comments</li>
-                        <li>Remove</li>
-                        <li>Edit</li>
-                    </ul>
-                </div>
-            </div>
-
-            <div className="oneBookRightContainer">
-
-            <div className="oneBookPercent">
-            <Progress type="circle" percent={oneBook.percent} showInfo={false} strokeColor="#379cf6" className="percentAntDesign"/>
-            <div className="oneBookPercentInnerAlign">
-                <span className="percentText">{oneBook.percent}%</span>
-                <span className="completedText">Completed</span>
-            </div>
-            </div>
-
-            <div className="oneBookCurrentChapter">
-                <span className="currentChapter">CURRENT CHAPTER</span>
-                <span className="numberChapter">{oneBook.chapter}</span>
-                <button className="updateButton"><span>UPDATE PROGRESS</span></button>
-            </div>
-            </div>
+    <div className="oneBookContainer">
+      <div className="oneBookInformation">
+        <div className="oneBookInformationText">
+
+          <span className="bookActionSpan">{genre}</span>
+          <h2 className="bookTitleH2">{title}</h2>
+          <span className="bookAuthorSpan">{author}</span>
+
+        </div>
+
+        <div className="oneBookInformationOptions">
+          <ul>
+            <li>Comments</li>
+            <li onClick={() => dispatch(removeBook(id))} aria-hidden="true">Remove</li>
+            <li>Edit</li>
+          </ul>
         </div>
+      </div>
+
+      <div className="oneBookRightContainer">
+
+        <div className="oneBookPercent">
+          <Progress type="circle" percent={percent} showInfo={false} strokeColor="#379cf6" className="percentAntDesign" />
+          <div className="oneBookPercentInnerAlign">
+            <span className="percentText">
+              {percent}
+              %
+            </span>
+            <span className="completedText">Completed</span>
+          </div>
+        </div>
+
+        <div className="oneBookCurrentChapter">
+          <span className="currentChapter">CURRENT CHAPTER</span>
+          <span className="numberChapter">{chapter}</span>
+          <button className="updateButton" type="button"><span>UPDATE PROGRESS</span></button>
+        </div>
+      </div>
+    </div>
 
   );
 };
-
+OneBook.propTypes = {
+  oneBook: PropTypes.element.isRequired,
+};
```

Comment:
- [OPTIONAL] There is a console error about prop. It seems the type of prop is wrong.

![image](https://user-images.githubusercontent.com/79876959/145260126-2ee8fabf-a77b-4114-b4f6-95e12ed8dcb1.png)

Reply:
Thanks, I just fixed this!